test/ibmpi_logger_test.go

```diff
@@ -1,11 +1,13 @@
-package ibmpisession
```

Comment:
Unit tests are meant to stay with the source package, let's move to them into a different package